transaction-status-client-types/src/lib.rs

```diff
@@ -226,12 +234,122 @@ impl From<&MessageAddressTableLookup> for UiAddressTableLookup {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct UiTransactionError(pub TransactionError);
```

Comment:
Start here for the RPC `UiTransactionError` and `UiTransactionResult` airgap change. These are essentially shims that ensure that the RPC serialization for transaction errors won't change its structure despite `TransactionError::InstructionError` having changed its structure.